react/client/public/index.html

```diff
@@ -48,6 +48,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
```

Comment:
Could you use icons via the npm package instead? https://www.npmjs.com/package/@material-ui/icons

Reply:
Got it!